assets/js/components/Dashboard/NewPaMessage/NewPaMessage.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import moment from "moment";
+import { Page } from "./types";
+import { Alert } from "Models/alert";
+
+import NewPaMessagePage from "./NewPaMessagePage";
+import AssociateAlertPage from "./AssociateAlertPage";
+
+const NewPaMessage = () => {
+  const [page, setPage] = useState<Page>(Page.NEW);
+
+  const now = moment();
+
+  const [associatedAlert, setAssociatedAlert] = useState<Alert>({} as Alert);
+  const [importLocations, setImportLocations] = useState<boolean>(true);
+  const [importMessage, setImportMessage] = useState<boolean>(true);
+  const [endWithEffectPeriod, setEndWithEffectPeriod] = useState<boolean>(true);
```

Comment:
Default should be `false`, right? I imagine this was hardcoded for testing so just want to make sure.

Reply:
Mary actually called out during QA that the checkboxes should default to selected

Reply:
That makes sense, but causes a problem when the page loads. Because it's true by default and there is no associated alert, the end date and time inputs are not visible.

Reply:
I added some more conditional rendering to let us keep the default as true but still render the date input. Let me know what you think of it

Reply:
@cmaddox5 going back to this one - I added another function `setAssociateAlertDefaults` for setting the default values of the import flags to `true` which I pass down to the button that takes the user to the associate alert page. If we just set the defaults in `NewPaMessage.tsx` as `false` and only set the flags to `true` once the user clicks navigate, then I think we can remove those additional conditions I added after your comment here.